src/webui/master/static/js/controllers.js

```diff
@@ -230,6 +230,34 @@
       _.each(framework.unreachable_tasks, setTaskMetadata);
       _.each(framework.completed_tasks, setTaskMetadata);
 
+      framework.running_tasks = 0;
```

Comment:
A TODO here for adding per-framework metrics so that we don't have to loop over the tasks would be great.